tests/legacy-ray_charles.e2e.spec.js

```diff
@@ -54,6 +54,12 @@ for (let i = 0; i < 15; i++) {
 }
 console.log('\nLoading archive:\n' + rayCharlesAllParts + '\n');
 
+/**
+ *  Run the tests
+ * @param {WebDriver} driver Selenium WebDriver object
+ * @param {array} modes Array of modes to run the tests in
+ * @returns {Promise<void>}
```

Comment:
You should add a comment at end of `@returns` line: ` A Promise for the completion of the tests`

Reply:
`return await driver.quit();` will actually return a `void` not `Promise`